go/apps/api/routes/openapi/handler.go

```diff
@@ -0,0 +1,31 @@
+package handler
```

Comment:
Added this handler for tools like scalar GUI that generate based off openapi spec.
Can remove if needed